kover-gradle-plugin/docs/configuring.md

```diff
@@ -64,10 +64,10 @@ koverReport {
               maxValue = 99
   
               // specify which units to measure coverage for
-              metric = kotlinx.kover.gradle.plugin.dsl.MetricType.LINE
+              coverageUnits = kotlinx.kover.gradle.plugin.dsl.MetricType.LINE
```

Comment:
After renaming `metric` to `coverageUnits`, enum `MetricType`  should also be renamed to something related, e.g. `UnitType` or `CoverageUnit`

Reply:
I think we can rename it